Privacy.md

```diff
@@ -0,0 +1,56 @@
+# BlitzMail Privacy Policy
+This is the developer's policy which is created to give details and informs you if your personal information is collected, used, disclosed and secured by the developer. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. <hr>
+
+## Permission and Access
+
+BlitzMail software requires the storage permission to function effectively and efficiently. Without this permission enabled, you might be unable to perform certain operations with the software.
+BlitzMail has some other capabilities which are functions the software can remotely control on its own.
+
+- **have full network access**: The app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+
+- **view network connections**: This is the capability of the app to view information about network connections such as which networks exist and are connected.
+
+- **prevent phone from sleeping**: When running, the app has the capability to prevent your device from sleeping.  
+
+- **run at startup**: This is the ability of the app to startup immediately by itself as soon as the system has finished booting.
+
+<hr>
+
+## Information and data collection
+
+The developer does not externally collect any information or data when the services of BlitzMail is utilized. When performing some activities with the software, personal data is not collected by the developer because all the information are locally stored on your device. 
+### How it works
+Users enter connection information for their email server and this server, of course, knows about the emails that are sent, also their clear-text, but this is normal and necessary. Otherwise, BlitzMail is not making any other internet connection and not sending any data anywhere.
+If an email can't be sent right away, it might be saved locally (on the user's device) until it can be sent.
+
+<hr>
+
+## Crash reports sending
+
+BlitzMail is stable and works fine but you may still experience few errors and crashes, information about these crashes are important to us so the issues can be fixed immediately and the software is running efficiently aga
```

Comment:
Is there some text missing at the end here?

Also, I am not sure we need crash report sending section as there are not crash reports being sent.